charts/project-origin-registry/values.yaml

```diff
@@ -95,6 +118,11 @@ immutableRecord:
       # key is the reference to the value within the secret
       key: ""
 
+# redis holds the configuration for the redis subchart
+redis:
+  # enabled specifies if the redis subchart should be enabled or not
+  enabled: true
+
```

Comment:
What would happen if the subchart was disabled? would Redis stop being deployed?

Reply:
yes, they they fallback to a inMemory cache per node, it is not recommended when running multiple nodes.